en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,470 @@
+
+2. Click **Settings**.
+
+3. Under **API Settings** click **Edit**.
+
+4. Toggle the **CORS Configuration** switch to enable the CORS configuration.
+
+5. Select the **Access Control Allow Credentials** checkbox.
+
+6. In the **Apply to Development** pane that opens on the right-hand side of the page, enter a meaningful message. Then click **Apply**.
+
+7. Click **Save**.
+
+#### Step 1.4.2: Publish the REST API
+
+We are now ready to publish the REST API. To do so, follow the steps given below:
+
+1. In the **Manage** tab, click **Lifecycle**.
+
+2. Click **Publish** to publish the REST API to the Developer Portal. External applications can subscribe to the API via the Developer Portal.
+
+3. To access the Developer Portal, click **Go to DevPortal**.
+
+    The readingList REST API will open in the Developer Portal.
+
+## Step 2: Consume the REST API
+
+You have published the readingList REST API to the Developer Portal where application developers can find it and subscribe their applications to it.
+
+In the previous steps, you played the role of a REST API developer and developed a REST API. In this step, you will play the role of the web application developer who will consume this REST API.
+
+To consume the `readingList` REST API, let's create an application, subscribe it to the REST API, generate keys, and invoke the API.
+
+### Step 2.1: Create an application
+
+An application in the Developer Portal is a logical representation of a physical application such as a mobile app, web app, device, etc.
+
+Let's create the application to consume the `readingList` REST API by following the steps given below:
+
+1. In the top menu of the Developer Portal, click **Applications**.
+
+2. Click **Create**.
+
+3. Enter a name for the application (for example, `readingListApp` and click **Create**.
+
+    Your Application will open on a separate page.
+
+### Step 2.2: Subscribe to the API
+
+To consume the REST API, the `readingListApp` application needs to subscribe to it. To subscribe your application to the API, follow the steps given below:
+
+1. In the left navigation menu, click **Subscriptions**.
+
+2. Click **Add APIs**.
+
+3. Find your REST API and click **Add**.
+
+Now your application has subscribed to the `readingList` REST API.
+
+
+### Step 2.3: Deploy a Web application and invoke the REST API
+
+At present, any user can invoke the `readingList` REST API via the `readingListApp` application (i.e., using its token) and update the same reading list. To allow multiple users to use the application and maintain unique reading lists, you need a front-end application that allows each user to log in with a unique user ID.
+
+In this step, let's deploy a pre-designed front-end application. This application is designed to personalize the readingList based on the user ID that it obtains from its identity provider. To enable the application to obtain the user ID, let's configure Asgardeo as the identity provider.
```

Comment:
Shall we mention what privilleges you need in Asgardeo to carry out the following tasks?

Reply:
Added an information note mentioning the privileges.

Reply:
This is not yet addressed.

Reply:
Added note on line 322.